src/components/m-table-body-row.js

```diff
@@ -17,12 +17,20 @@ export default class MTableBodyRow extends React.Component {
       .sort((a, b) => a.tableData.columnOrder - b.tableData.columnOrder)
       .map((columnDef, index) => {
         const value = this.props.getFieldValue(this.props.data, columnDef);
+        const indentStyle = (this.props.options.indentFirstDataCell && index == 0)?(
+          size === 'medium' ? {
+            paddingLeft: this.props.level * 9
```

Comment:
I dont think this is the best way to do it.

Consider a nesting 3-4 layers deep. you will want the indentation to still work